timed/reports/management/commands/notify_supervisors_shorttime.py

```diff
@@ -74,10 +74,14 @@ def _get_supervisees_with_shorttime(self, start, end, ratio):
         Get supervisees which reported less hours than they should have.
 
         :return: dict mapping all supervisees with shorttime with tuple of
-                 reported, expected, balance and actual ratio.
+                 reported, expected, balance, actual ratio and current balance.
```

Comment:
I think a nested dict would be more useful than a tuple here.

Also note the docstring of _notify_supervisors() further below does not match this list

Reply:
there is no built in way to access dicts in django templates - that's why I have chosen to use a tuple.

Reply:
You can access dict keys just like object properties

Reply:
Changed.